framework/db/Query.php

```diff
@@ -475,11 +476,8 @@ protected function queryScalar($selectExpression, $db)
             $this->limit = null;
             $this->offset = null;
 
-            $e = null;
             try {
                 $command = $this->createCommand($db);
-            } catch (\Exception $e) {
```

Comment:
That would break things on older PHP versions where there's no Throwable.

Reply:
reverted